frontend/src/components/IngredientsList.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components/macro";
+import IngredientItem from "./IngredientItem";
+
+export default function IngredientsList({ ingredientsList }) {
+  console.log(ingredientsList);
+  return (
+    <Wrapper>
+      {ingredientsList.map((ingredient, index) => (
+        <IngredientItem key={index} ingredient={ingredient} />
```

Comment:
```suggestion
        <IngredientItem key={ingredient.id} ingredient={ingredient} />
```

Reply:
If I change this line, my code isn't working.